src/Frontend/Pages/GameLandingPage.tsx

```diff
@@ -290,11 +289,26 @@ export function GameLandingPage({ match, location }: RouteComponentProps<{ contr
       terminal.current?.println(`Found ${accounts.length} accounts on this device.`);
       terminal.current?.println(``);
 
-      if (accounts.length > 0) {
-        terminal.current?.print('(a) ', TerminalTextStyle.Sub);
-        terminal.current?.println('Login with existing account.');
-      }
+      try {
+        const balances = await loadBalances(accounts.map((a) => a.address));
 
+        accounts.forEach((account, i) => {
+          terminal.current?.print(`(${i + 1}): `, TerminalTextStyle.Sub);
+          terminal.current?.print(`${account.address} `);
+          terminal.current?.println(
+            balances[i].toFixed(2) + ' xDAi',
+            balances[i] < 0.01 ? TerminalTextStyle.Red : TerminalTextStyle.Green
```

Comment:
```suggestion
            balances[i] < 0.001 ? TerminalTextStyle.Red : TerminalTextStyle.Green
```
We will make this a constant eventually.